packages/core/src/renderer/components/dropdown/dropdown.tsx

```diff
@@ -31,7 +34,7 @@ export function Dropdown(props: DropdownProps) {
         close={toggle}
         open={toggle}
       >
-        {React.Children.toArray(children)}
+        {toSafeReactChildrenArray(children)}
```

Comment:
How is this different than`{children}` since we are not manipulating children at all? Notice that overall manipulating children is considered bad practice.

Reply:
```tsx
<Dropdown>{[<SomeComponent key={1} />, <SomeComponent key={2} />]}</Dropdown>
```

Works fine with `{children}`

Reply:
Sounds good